docs/platform-parts/cauldron.md

```diff
@@ -135,7 +135,7 @@ $ ern cauldron add <native-app-descriptor>
 // for example
 // ern cauldron add nativeapp MyApp:ios:0.0.1
 ```
-
+The *complete native application descriptor* is a string with format `[nativeAppName]:[platform]:[version]` 
```

Comment:
Same remark as before, no need to precise format here, its linked to the glossary, better to keep it centralized there.
Also no need to put it in italic, there is already format processing because of the auto linking with glossary that underlines it and put it in blue by default. we can put all glossary terms links in italic directly in the css styling for these links if we want to put them in italic (which I don't think is needed anyway, just a color difference is nice enough)